docs/api-reference/tasks-api.md

```diff
@@ -25,77 +25,1448 @@ sidebar_label: Tasks
 
 This document describes the API endpoints for task retrieval, submission, and deletion for Apache Druid.
 
-## Tasks
-
 Note that all _interval_ URL parameters are ISO 8601 strings delimited by a `_` instead of a `/`
 as in `2016-06-27_2016-06-28`.
 
-`GET /druid/indexer/v1/tasks`
+## Task information and retrieval 
+
+### Get an array of tasks
+
+#### URL
+<code class="getAPI">GET</code> `/druid/indexer/v1/tasks`
+
+This endpoint retrieves an array of all task objects currently running or executed in the current Druid cluster. It provides information about each task such as its task id, task status, associated data source, and other metadata. It supports a set of optional query parameters to filter results. 
```

Comment:
What does "or executed" mean, completed? Would it be accurate to say "all tasks objects in the Druid cluster"?

Also not sure what "current" Druid cluster means. (and ditto for all the requests that mention it)